dali/operators/reader/loader/video/frames_decoder_gpu.cc

```diff
@@ -134,14 +134,15 @@ FramesDecoderGpu::FramesDecoderGpu(const std::string &filename, cudaStream_t str
 }
 
 FramesDecoderGpu::FramesDecoderGpu(
-  const char *memory_file, int memory_file_size, cudaStream_t stream) :
-  FramesDecoder(memory_file, memory_file_size),
+  const char *memory_file, int memory_file_size, cudaStream_t stream, bool build_index) :
+  FramesDecoder(memory_file, memory_file_size, build_index),
   frame_buffer_(num_decode_surfaces_),
   stream_(stream) {
   InitGpuDecoder();
 }
 
-int FramesDecoderGpu::ProcessPictureDecode(void *user_data, CUVIDPICPARAMS *picture_params) {
+int FramesDecoderGpu::ProcessPictureDecode(
+  void *user_data, CUVIDPICPARAMS *picture_params) {
```

Comment:
Why? It doesn't need line breaks to fit in our line length limit.

Reply:
Done